src/market/directory.rs

```diff
@@ -81,8 +122,8 @@ pub async fn post_maker_address_to_directory_servers(
     network: Network,
     address: &str,
 ) -> Result<u64, DirectoryServerError> {
-    let proxy =
-        reqwest::Proxy::all(format!("socks5h://{}", TOR_ADDR)).expect("tor proxy should be there");
+    let proxy = reqwest::Proxy::all(format!("socks5h://{}", TOR_SOCKS_ADDR))
+        .expect("tor proxy should be there");
```

Comment:
This enitre function should be in maker API. Could have a simpler name like `post_addr_to_dns`.